values.yaml

```diff
@@ -535,27 +527,24 @@ notary:
   # They will be generated if not set.
   secretName: ""
 
+postgresql:
+  enabled: true
+  postgresqlPostgresPassword: changeit
+  postgresqlUsername: harbor
+  postgresqlPassword: changeit
+  postgresqlDatabase: registry
+  initdbScripts:
+    db-init.sql: |
+      CREATE DATABASE clair;
+      CREATE DATABASE notary_server;
+      CREATE DATABASE notary_signer;
+      GRANT ALL PRIVILEGES ON DATABASE clair TO harbor;
+      GRANT ALL PRIVILEGES ON DATABASE notary_server TO harbor;
+      GRANT ALL PRIVILEGES ON DATABASE notary_signer TO harbor;
```

Comment:
why not have this in the databse sub-section?

Reply:
Because this chart variation depends on the postrgresql chart as listed in requirements.yaml.
Any values affecting the sub-chart will need to be prefixed by the chart name.